app/src/main/java/com/spendster/presentation/signUp/SignUpView.java

```diff
@@ -3,11 +3,9 @@
 public interface SignUpView {
```

Comment:
@SimonMaha this interface should be dropped into two: "AuthView" and "SignUpView"